beacon_chain/utils/bls/src/lib.rs

```diff
@@ -10,16 +10,24 @@ pub use self::bls_aggregates::Signature;
 
 pub const BLS_AGG_SIG_BYTE_SIZE: usize = 97;
 
-use hashing::proof_of_possession_hash;
+use hashing::canonical_hash;
+use std::default::Default;
+
+fn extend_if_needed(hash: &mut Vec<u8>) {
+    // NOTE: bls_aggregates crate demands 48 bytes, this may be removed as we get closer to production
+    hash.resize(48, Default::default())
```

Comment:
This means that `hash` can be more than 48 bytes. I don't _think_ it'll be a problem but probably worth checking out (unless you already have).

Reply:
unless i'm reading the docs incorrectly, it seems like we should only be passing a value for `hash` s.t. `hash.len()` is 32. and then we resize to 48 so if `hash.len()` is bigger it will shrink and if `hash.len()` is smaller it will extend w/ `0x00`. so hash should not ever be more than 48 bytes -- am i missing something?

Reply:
Oh sorry, you're totally right. Brain failure on my end.